Docker/dev.Dockerfile

```diff
@@ -7,6 +7,7 @@
 # code. The next two layers use the dotnet/aspnet image to run the built code.
 # The extra layers in the middle support caching of base layers.
 
+#tag sdk:8.0-alpine
 FROM mcr.microsoft.com/dotnet/sdk:8.0.203-alpine3.18@sha256:2a8dca3af111071172b1629c12eefaeca0d6c2954887c4489195771c9e90833c as buildBase
 
 FROM buildbase as build
```

Comment:
Looks like we didn't need this extra layer - since it is unchanged from `buildBase` - but there's also no harm to it.